components/chart/canvas/Module.tsx

```diff
@@ -14,12 +15,16 @@ const ModuleTypeToClassName = {
 
 function Module({ module, selectModule, isSelected }: ModuleProps) {
   const moduleName = module.title || ModuleInfo.getModuleName(module.type);
-
+  if (module.x == 0 && module.y == 0 || module.x == 0 || module.y == 0) {
+    module.x += MODULE_X;
+    module.y += MODULE_y;
+  }
```

Comment:
I think it would be better to add the defaults padding when rendering the nodes, not here

Reply:
done